### PR TITLE
http: show neighbors address list

### PR DESCRIPTION
**Pull request by dylenfu**

### CODE REVIEW

File: http/nodeinfo/server.go [hunk 1]
```diff
@@ -111,11 +113,24 @@ func viewHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func peerAddressListHandler(w http.ResponseWriter, r *http.Request) {
+	list := make([]string, 0)
+
+	nbrs := node.GetNeighbors()
+	for _, v := range nbrs {
+		list = append(list, v.GetAddr())
```
Comment (Honglei-Cong) [inline http/nodeinfo/server.go]: suggest updating PeerInfo.String(), and show all peer info.

File: http/nodeinfo/server.go [hunk 2]
```diff
@@ -111,11 +113,24 @@ func viewHandler(w http.ResponseWriter, r *http.Request) {
...
+	bz, _ := json.Marshal(list)
+	io.WriteString(w, string(bz))
+}
+
 func StartServer(n p2p.P2P) {
 	node = n
 	port := int(config.DefConfig.P2PNode.HttpInfoPort)
 
 	http.HandleFunc("/info", viewHandler)
+	http.HandleFunc("/info/neighborAddress", peerAddressListHandler)
```
Comment (Honglei-Cong) [inline http/nodeinfo/server.go]: change the API to `/info/neighbors`, 
dont just dump neighbor IP addresses.

Comment (dylenfu) [reply http/nodeinfo/server.go]: already modified

File: http/nodeinfo/server.go (hunk 1 above)
Comment (dylenfu) [reply http/nodeinfo/server.go]: ok

**Pull request closed without merging**